dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlannerBuilder.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Planning.Sequential;
+
+/// <summary>
+/// Builder for <see cref="ISequentialPlanner"/>.
+/// </summary>
+public class SequentialPlannerBuilder
+{
+    /// <summary>
+    /// Initialize a new instance of the <see cref="SequentialPlannerBuilder"/> class.
+    /// </summary>
+    /// <param name="kernel">The semantic kernel instance.</param>
+    public SequentialPlannerBuilder(IKernel kernel)
+    {
+        this._kernel = kernel;
+    }
+
+    /// <summary>
+    /// Build a new planner instance using the settings passed so far.
+    /// </summary>
+    /// <returns>Planner instance.</returns>
+    public ISequentialPlanner Build()
+    {
+        ISequentialPlanner instance = new SequentialPlanner(this._kernel, this._config, this.prompt);
+
+        instance = new InstrumentedSequentialPlanner(instance, this._logger);
```

Comment:
This will always replace the instance above

Reply:
That's desired behavior of decoration. We can also do:
```csharp
return new InstrumentedSequentialPlanner(new SequentialPlanner(this._kernel, this._config, this.prompt), this._logger);
```

Reply:
Oh duh. Sorry, I didn't see that it was actually wrapping the above instance. I thought it was just overwriting. Thanks for the clarification!